.travis.yml

```diff
@@ -1,19 +1,18 @@
 language: php
 
 php:
-  - 5.6
   - 7.0
-  - 7.1
+  - 7.3
```

Comment:
Not sure if there is much value in prehaps adding 7.1 and 7.2?

Reply:
I was deliberately only listing lowest and highest - I thought that would be enough.